src/main/java/com/freenow/apis/phrase/api/localedownload/PhraseLocaleDownloadAPI.java

```diff
@@ -96,15 +109,21 @@ public byte[] downloadLocale(final String projectId, final String branch, final
                 parameters.add(new BasicNameValuePair(PARAMETER_BRANCH, branch));
             }
 
+            if (tags != null && !tags.trim().isEmpty()) {
+                parameters.add(new BasicNameValuePair(PARAMETER_TAGS, tags));
+            }
+
             final URIBuilder builder = createUriBuilder(requestPath, parameters);
 
-            final HttpEntity<Object> requestEntity = createHttpEntity(requestPath);
+            final String requestPathWithTags = requestPath.concat(getNonNullString(tags));
```

Comment:
Why it should be a concat instead of adding this logic to createDownloadLocaleRequestPath method?

Reply:
The mentioned method creates the real path to request; `requestPath.concat(getNonNullString(tags))` creates the key for ETag cache. I renamed the variable to make it more clear.